src/20231106_AaveV3Ethereum_AaveFundingUpdates/AaveFundingUpdates.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Aave Funding Updates"
+author: "TokenLogic, efecarranza.eth"
+discussions: "https://governance.aave.com/t/arfc-aave-funding-update/15194"
+---
+
+## Simple Summary
+
+This AIP aims to consolidate Aave DAO's stablecoin holdings. It is divided into different parts that are detailed below. Part one is depositing DAI into aEthDAI, part two is migrating aDAI and aUSDT into aEthDAI and aEthUSDT, part three bridges assets from Polygon to Mainnet and part four swaps some holdings into GHO.
```

Comment:
summary is not really aligned with the specification and code

Reply:
updating